File: panel/src/main/java/com/effective/android/panel/utils/cutShort/DeviceCutShort.java
package com.effective.android.panel.utils.cutShort;

import android.content.Context;
import android.view.View;

public interface DeviceCutShort {

    boolean hasCutShort(Context context);

    boolean isCusShortVisible(Context context);

    int getCurrentCutShortHeight(View view);
}

File: panel-kotlin/src/main/java/com/effective/android/panel/utils/cutShort/DeviceCutShort.kt
package com.effective.android.panel.utils.cutShort

import android.content.Context
import android.view.View

interface DeviceCutShort {
    fun hasCutShort(context: Context): Boolean
    fun isCusShortVisible(context: Context): Boolean
    fun getCurrentCutShortHeight(view: View): Int
}